lib/charms/kubernetes_charm_libraries/v0/multus.py

```diff
@@ -530,6 +530,8 @@ def _configure_network_attachment_definitions(self):
         2. Goes through the list of NetworkAttachmentDefinitions to create and create them all
         """
         network_attachment_definitions_to_create = self.network_attachment_definitions_func()
+        if not network_attachment_definitions_to_create:
```

Comment:
We should not make this change here, here we should only `fetch-lib`. The actual maintainer of this lib is [this project](https://github.com/canonical/kubernetes-charm-libraries/blob/main/lib/charms/kubernetes_charm_libraries/v0/multus.py#L105) whose lib version in `5`.

Reply:
Done